app/views/posts/show.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Posts</h1>
-<p>Show one post for a given user</p>
+<main>
+<section class="postscontainer">
+<div class="postcomment">
+<h4>Post<%= @post.id%><h4>
+  <h4><%= @post.title %></h4>
+  <p class="commentcounter">Comments: <%= @post.comments_counter || 0%>, Likes: <%= @post.likes_counter || 0%></p>
+  <p><%= @post.text %></p>
+  </div>
+  <%= render partial: 'comment', locals: {post: @post, comments: @post.comments}%>
```

Comment:
- [ ] According to the previous reviewer's suggestion, please include the username of the author in the post show page
### Requirement
![image](https://user-images.githubusercontent.com/46839548/175132481-6df24fc4-22a7-45c7-a9b8-d38de5c7c529.png)
### Your design
![image](https://user-images.githubusercontent.com/46839548/175132663-15adb87d-ef4d-4a88-a574-9773e9bf8eac.png)